apps/desktop/src/renderer/styles/app.scss

```
@@ -1,4 +1,4 @@
-@import "tailwindcss";
+@import url("tailwindcss");
```

Stylelint (17.6.0)

[error] Expected "url("tailwindcss")" to be ""tailwindcss"" (import-notation)

(import-notation)